Lib/test/support/strace_helper.py

```diff
@@ -72,6 +72,27 @@ def sections(self):
         return sections
 
 
+def filter_memory(syscalls):
+    """Filter out memory allocation calls from File I/O calls.
+
+    Some calls (mmap, munmap, etc) can be used on files or to just get a block
+    of memory. Use this function to filter out the memory related calls from
+    other calls."""
+
+    def _filter(call):
```

Comment:
Can you move this function at the module level? Maybe rename it to _filter_memory_call().

Reply:
Can move, my thought initially was "Track the address mmap returns through munmap" and the set/dictionary of "known addresses" would be in the outer scope, but I don't think `mmap` in `io` is likely (there's the `mmap` module if it's better for a particular use case)